airbyte-integrations/connectors/destination-iceberg/build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
         exclude(group: 'org.apache.hadoop', module: 'hadoop-common')
     }
     implementation 'org.apache.iceberg:iceberg-spark-runtime-3.3_2.13:1.3.0'
+    implementation 'org.projectnessie.nessie-integrations:nessie-spark-extensions-3.3_2.13:0.59.0'
```

Comment:
nit: maybe add a comment that this version needs to be kept in sync with the Nessie version in Iceberg?

Reply:
I think we should remove the ref link there. Because 1.4.0 uses gradle version catalog and the file names are different. 
Just mentioning that versions need to be in sync is enough.